variable/technology/technologies.yaml

```diff
@@ -186,6 +189,74 @@ Capital Cost|Liquids|Oil:
    description: Capital cost of a new oil refining plant
    unit: US$2010/kW
 
+Constant Variable Cost|Electricity|Coal:
+   description: Constant term (intercept) of the heat rate straight line. Cost of the power generation unit who belongs to the specified power
+                plant category.
+   unit: Mcal/h
+
+Constant Variable Cost|Electricity|Gas:
+   description: Constant term (intercept) of the heat rate straight line. Cost of the power generation unit who belongs to the specified power
+                plant category.
+   unit: Mcal/h
+
+Constant Variable Cost|Electricity|Nuclear:
+   description: Constant term (intercept) of the heat rate straight line. Cost of the power generation unit who belongs to the specified power
+                plant category.
+   unit: Mcal/h
+
+Constant Variable Cost|Electricity|Oil:
+   description: Constant term (intercept) of the heat rate straight line. Cost of the power generation unit who belongs to the specified power
+                plant category.
+   unit: Mcal/h
+
+Energy Charge|Electricity|Energy Storage System:
+   description: Energy charge by ESS for each load level
+   unit:  TWh
+
+Energy Not Serve Cost|Electricity:
+   description: Cost of energy not served. Cost of load curtailment. Value of Lost Load (VoLL)
+   unit:  €/MWh
+
+Energy Not Serve|Electricity:
+   description: Energy not served by node
+   unit:  GWh
+
+Fixed Investment Cost|Electricity|Transmission|Line:
+   description: Overnight investment (capital) cost in transmission lines of the power network
+   unit: M€
+
+Fixed Charge Rate|Electricity|Transmission|Line:
+   description: Fixed charge rate to annualize the overnight investment cost
+   unit: p.u.
+
+Fuel Cost|Electricity|Coal:
```

Comment:
Isn't that the same as `Price|Primary Energy|Coal`?

Reply:
Yeah. For this case, is it possible to add or request the addition of the unit [M€/Mcal] to variable `Price|Primary Energy|...`?
Then, my variables will be:

> `linear   term variable cost|...   [€/GWh] = Linear Variable Cost|...  [Mcal/GWh] * Price|Primary Energy|... [€/Mcal] + Operation and Maintenance Variable Cost|... [€/MWh] * 1e-3`

> `constant term variable cost|...   [€/h]   = Variable Cost (Intercept)|... [Mcal/h] * Price|Primary Energy [€/Mcal]`

Reply:
Correction:

> `linear term variable cost|... [€/GWh] = Variable Cost (slope)|... [Mcal/GWh] * Price|Primary Energy|... [€/Mcal] + Operation and Maintenance Variable Cost|... [€/MWh] * 1e-3`